src/components/organisms/HomePanel/HomePanel.tsx

```diff
@@ -71,174 +72,218 @@ const HomePanel = ({
       4: "w-4",
     },
   };
+  /*
+   * TUTORIAL CONFIG
+   */
+  const tourConfigMine = [
+    {
+      target: ".step1-total-number",
+      content: t("onboarding.demo.home.step1"),
+      title: t("onboarding.demo.home.titlestep1"),
+    },
+    {
+      target: ".step2-5bestskills",
+      content: t("onboarding.demo.home.step2"),
+    },
+    {
+      target: ".step3-graph",
+      content: t("onboarding.demo.home.step3"),
+      title: t("onboarding.demo.home.titlestep3"),
+    },
+  ];
+
+  const tourConfigNoSkill = [
+    {
+      target: ".step3-graph",
+      content: t("onboarding.demo.zenika.step3"),
+      title: t("onboarding.demo.zenika.titlestep3"),
+    },
+  ];
```

Comment:
Maybe import all tour step from another file. This avoid overloading a file component